### PR TITLE
Release notes 1.1

### LINKED ISSUE #4570: Knative 1.1 release notes blog post
Consolidate all components and write a blog post similar to https://knative.dev/blog/releases/announcing-knative-v1-0-release/

### PR DESCRIPTION
**Pull request by csantanapr**
Fixes #4570

### CODE REVIEW

File: blog/docs/releases/announcing-knative-v1-1-release.md [hunk 1]
```diff
@@ -0,0 +1,166 @@
...
+
+### Eventing v1.1
+
+<!-- Original notes are here: https://github.com/knative/eventing/releases/tag/knative-v1.1.0 -->
+
+#### 💫 New Features & Changes
+
+- New experimental-feature "delivery-retryafter" flag allows use of "DeliverySpec.retryAfter" to configure handling of Retry-After headers in 429 / 503 responses. See https://github.com/knative/docs/blob/main/docs/eventing/experimental-features.md ([#5813](https://github.com/knative/eventing/pull/5813)
+- All core Knative Eventing Pods should now be able to run in the restricted pod security standard profile. ([#5863](https://github.com/knative/eventing/pull/5863)
+- Triggers now include a CloudEvents Subscriptions API compatible filters field as an experimental feature. ([#5715](https://github.com/knative/eventing/pull/5715)
```
Comment (psschwei) [inline blog/docs/releases/announcing-knative-v1-1-release.md:65]: Each line here needs a closing `)`

File: blog/docs/releases/announcing-knative-v1-1-release.md [hunk 2]
```diff
@@ -0,0 +1,166 @@
...
+
+#### 🐞 Bug Fixes
+
+- The Sequence type accepts any Channel spec field in the spec.channelTemplate.spec field. ([#5955](https://github.com/knative/eventing/pull/5955)
+- Upgrade dependencies that contains vulnerabilities in snyk DB ([#5889](https://github.com/knative/eventing/pull/5889)
```
Comment (psschwei) [inline blog/docs/releases/announcing-knative-v1-1-release.md:70]: closing `)`

File: blog/docs/releases/announcing-knative-v1-1-release.md [hunk 3]
```diff
@@ -0,0 +1,166 @@
...
+
+#### Apache Kafka Broker v1.1
+
+<!-- Original notes are here: https://github.com/knative-sandbox/eventing-kafka-broker/releases/tag/knative-v1.1.0 -->
+
+#### 🚨 Breaking or Notable Changes
+
+- Add a new implementation of the KafkaSource API ([#1415](https://github.com/knative-sandbox/eventing-kafka-broker/pull/1415)
+- Kafka Broker event delivery is hundreds of times faster ([#1405](https://github.com/knative-sandbox/eventing-kafka-broker/pull/1405)
+- broker.spec.config is now required. ([#1555](https://github.com/knative-sandbox/eventing-kafka-broker/pull/1555)
```
Comment (psschwei) [inline blog/docs/releases/announcing-knative-v1-1-release.md:82]: closing `)`

File: blog/docs/releases/announcing-knative-v1-1-release.md [hunk 4]
```diff
@@ -0,0 +1,166 @@
...
+
+#### 💫 New Features & Changes
+
+- The kafka-controller deployment emits probe requests against the data plane (kafka-sink-receiver and kafka-broker-receiver) to determine Kafka Broker and KafkaSink readiness. ([#1495](https://github.com/knative-sandbox/eventing-kafka-broker/pull/1495)
+- You can now configure which header format(b3 multi header, b3 single header, and w3c trace-context) to be used while using Zipkin backend ([#1546](https://github.com/knative-sandbox/eventing-kafka-broker/pull/1546)
+- Handle non-retryable HTTP status codes as reported in the spec:
+For more information: https://github.com/knative/specs/blob/c348f501de9eb998b4fd010c54d9127033ee41be/specs/eventing/data-plane.md#event-acknowledgement-and-delivery-retry ([#1574](https://github.com/knative-sandbox/eventing-kafka-broker/pull/1574)
```
Comment (psschwei) [inline blog/docs/releases/announcing-knative-v1-1-release.md:89]: closing `)`

File: blog/docs/releases/announcing-knative-v1-1-release.md [hunk 5]
```diff
@@ -0,0 +1,166 @@
...
+<!-- Original notes are here: https://github.com/knative-sandbox/eventing-rabbitmq/releases/tag/knative-v1.1.0 -->
+
+
+#### 💫 New Features & Changes
+
+- Adds tracing with opencensus to the dispatcher and ingress ([#370](https://github.com/knative-sandbox/eventing-rabbitmq/pull/370)
+- RabbitMQ source now can use predeclared queues instead of creating new ones ([#493](https://github.com/knative-sandbox/eventing-rabbitmq/pull/493)
+- Now the source Adapter processes messages concurrently depending on the channel_config.prefetch_count argument (default to 1) ([#522](https://github.com/knative-sandbox/eventing-rabbitmq/pull/522)
+- Do not expose the broker class that the controller operates on as an environment variable. ([#512](https://github.com/knative-sandbox/eventing-rabbitmq/pull/512)
+- The RabbitMQ Source now translates RabbitMQ messages according to the RabbitMQ Protocol Binding Spec for CloudEvents ([#475](https://github.com/knative-sandbox/eventing-rabbitmq/pull/475)
```
Comment (psschwei) [inline blog/docs/releases/announcing-knative-v1-1-release.md:103]: closing `)`

File: blog/docs/releases/announcing-knative-v1-1-release.md [hunk 6]
```diff
@@ -0,0 +1,166 @@
...
+  - Falls back to RabbitMQ message content type if it is not set on the CloudEvent data or headers.
+  - Avoids re-wrapping RabbitMQ messages that are already in the CloudEvent format
```
Comment (psschwei) [inline blog/docs/releases/announcing-knative-v1-1-release.md:105]: Were there PRs for these two?

File: blog/docs/releases/announcing-knative-v1-1-release.md [hunk 7]
```diff
@@ -0,0 +1,166 @@
...
+
+#### 🐞 Bug Fixes
+
+- The trigger dispatcher will now only retry deliveries when response status codes status are 5XX, 404, 409, 429, or -1. ([#486](https://github.com/knative-sandbox/eventing-rabbitmq/pull/486)
```
Comment (psschwei) [inline blog/docs/releases/announcing-knative-v1-1-release.md:109]:
```suggestion
- The trigger dispatcher will now only retry deliveries when response status codes status are 5XX, 404, 409, 429, or -1. ([#486](https://github.com/knative-sandbox/eventing-rabbitmq/pull/486))
```

File: blog/docs/releases/announcing-knative-v1-1-release.md [hunk 8]
```diff
@@ -0,0 +1,166 @@
...
+
+- Serving now has a global `min-scale` configuration
+- Eventing brings more experimental features
+
+
+### Serving v1.1
+
+<!-- Original notes are here: https://github.com/knative/serving/releases/tag/knative-v1.1.0 -->
+
+#### 💫 New Features & Changes
```
Comment (psschwei) [inline blog/docs/releases/announcing-knative-v1-1-release.md:41]: nit: Some of the bullet points end with periods but others don't (not sure what the official style guide recommends)

File: blog/docs/releases/announcing-knative-v1-1-release.md (hunk 6 above)
Comment (csantanapr) [reply blog/docs/releases/announcing-knative-v1-1-release.md:105]: They are 2 bullets under the previews line item, they part of the same PR above

File: blog/docs/releases/announcing-knative-v1-1-release.md (hunk 8 above)
Comment (csantanapr) [reply blog/docs/releases/announcing-knative-v1-1-release.md:41]: I didn't follow a guideline, I just copy-paste the release notes verbatim from the GitHub releases pages on each repo. 😄

File: blog/docs/releases/announcing-knative-v1-1-release.md (hunk 1 above)
Comment (csantanapr) [reply blog/docs/releases/announcing-knative-v1-1-release.md:65]: Good catch an all this formatting items 🙏

File: blog/docs/releases/announcing-knative-v1-1-release.md (hunk 8 above)
Comment (csantanapr) [reply blog/docs/releases/announcing-knative-v1-1-release.md:41]: went without a period

**Comment by csantanapr** [comment]
Hi @psschwei I addressed your comments, PTAL

File: blog/docs/releases/announcing-knative-v1-1-release.md [hunk 9]
```diff
@@ -0,0 +1,172 @@
...
+- The activator optimisation which directly probes the queue proxy for readiness rather than waiting for Kubernetes to report readiness is now disabled when exec probes are used (since queue proxy cannot execute these probes on the user container's behalf) ([#12250](https://github.com/knative/serving/pull/12250))
+- Use pkg/drain in queue proxy ([#12033](https://github.com/knative/serving/pull/12033))
+
+### Eventing v1.1
+
+<!-- Original notes are here: https://github.com/knative/eventing/releases/tag/knative-v1.1.0 -->
+
+#### 💫 New Features & Changes
+
+- New experimental-feature "delivery-retryafter" flag allows use of "DeliverySpec.retryAfter" to configure handling of Retry-After headers in 429 / 503 responses. See https://github.com/knative/docs/blob/main/docs/eventing/experimental-features.md ([#5813](https://github.com/knative/eventing/pull/5813))
```
Comment (psschwei) [inline blog/docs/releases/announcing-knative-v1-1-release.md:63]:
```suggestion
- New experimental-feature "delivery-retryafter" flag allows use of "DeliverySpec.retryAfter" to configure handling of Retry-After headers in 429 / 503 responses. See [https://github.com/knative/docs/blob/main/docs/eventing/experimental-features.md](experimental-features.md) ([#5813](https://github.com/knative/eventing/pull/5813))
```

File: blog/docs/releases/announcing-knative-v1-1-release.md [hunk 10]
```diff
@@ -0,0 +1,172 @@
...
+
+- Add cluster wide default min-scale ([#12290](https://github.com/knative/serving/pull/12290))
+- HPA autoscaler stabilization window can be set from autoscaling window annotation ([#12286](https://github.com/knative/serving/pull/12286))
+- Metrics with names other than "cpu" or "memory" are allowed as pod custom metrics ([#12277](https://github.com/knative/serving/pull/12277))
+
+#### 🐞 Bug Fixes
+
+- Allow setting TCP and HTTP port to be equal to containerPort on readiness and liveness probes ([#12225](https://github.com/knative/serving/pull/12225))
+- Drops the unix socket listener from Queue Proxy, since it is no longer used ([#12298](https://github.com/knative/serving/pull/12298))
+- Generated HPAs are now v2beta2.
```
Comment (psschwei) [inline blog/docs/releases/announcing-knative-v1-1-release.md:51]:
```suggestion
- Generated HPAs are now v2beta2
```

File: blog/docs/releases/announcing-knative-v1-1-release.md [hunk 11]
```diff
@@ -0,0 +1,172 @@
...
+- Adds tracing with opencensus to the dispatcher and ingress ([#370](https://github.com/knative-sandbox/eventing-rabbitmq/pull/370))
+- Now the source Adapter processes messages concurrently depending on the channel_config.prefetch_count argument (default to 1) ([#522](https://github.com/knative-sandbox/eventing-rabbitmq/pull/522))
+- Adds tracing with opencensus to the dispatcher and ingress ([#370](https://github.com/knative-sandbox/eventing-rabbitmq/pull/370))
+- Do not expose the broker class that the controller operates on as an environment variable ([#512](https://github.com/knative-sandbox/eventing-rabbitmq/pull/512))
+- Adds tracing with opencensus to the dispatcher and ingress ([#370](https://github.com/knative-sandbox/eventing-rabbitmq/pull/370))
+- The RabbitMQ Source now translates RabbitMQ messages according to the RabbitMQ Protocol Binding Spec for CloudEvents ([#475](https://github.com/knative-sandbox/eventing-rabbitmq/pull/475))
+  - Falls back to RabbitMQ message content type if it is not set on the CloudEvent data or headers.
+  - Avoids re-wrapping RabbitMQ messages that are already in the CloudEvent format
```
Comment (psschwei) [inline blog/docs/releases/announcing-knative-v1-1-release.md:111]: Think the indentation needs to be a bit more (it's not showing up as indented in the preview). 
```suggestion
    - Falls back to RabbitMQ message content type if it is not set on the CloudEvent data or headers.
    - Avoids re-wrapping RabbitMQ messages that are already in the CloudEvent format
```

**Review by psschwei — COMMENTED** [review]
A couple of minor formatting things...

File: blog/docs/releases/announcing-knative-v1-1-release.md (hunk 11 above)
Comment (csantanapr) [reply blog/docs/releases/announcing-knative-v1-1-release.md:111]: yeah it doesn't ident

Comment (csantanapr) [reply blog/docs/releases/announcing-knative-v1-1-release.md:111]: good catch

Comment (csantanapr) [reply blog/docs/releases/announcing-knative-v1-1-release.md]: Now it looks correct with 4 spaces
image

**Comment by csantanapr** [comment]
@psschwei address your last comments, PTAL

File: blog/docs/releases/announcing-knative-v1-1-release.md [hunk 12]
```diff
@@ -0,0 +1,172 @@
...
+- The activator optimisation which directly probes the queue proxy for readiness rather than waiting for Kubernetes to report readiness is now disabled when exec probes are used (since queue proxy cannot execute these probes on the user container's behalf) ([#12250](https://github.com/knative/serving/pull/12250))
+- Use pkg/drain in queue proxy ([#12033](https://github.com/knative/serving/pull/12033))
+
+### Eventing v1.1
+
+<!-- Original notes are here: https://github.com/knative/eventing/releases/tag/knative-v1.1.0 -->
+
+#### 💫 New Features & Changes
+
+- New experimental-feature "delivery-retryafter" flag allows use of "DeliverySpec.retryAfter" to configure handling of Retry-After headers in 429 / 503 responses. See [experimental-features.md](https://github.com/knative/docs/blob/main/docs/eventing/experimental-features.md) ([#5813](https://github.com/knative/eventing/pull/5813))
```
Comment (psschwei) [inline blog/docs/releases/announcing-knative-v1-1-release.md:63]: This link is broken 

```suggestion
- New experimental-feature "delivery-retryafter" flag allows use of "DeliverySpec.retryAfter" to configure handling of Retry-After headers in 429 / 503 responses. See [experimental-features](https://github.com/knative/docs/blob/main/docs/eventing/experimental-features) ([#5813](https://github.com/knative/eventing/pull/5813))
```

**Review by psschwei — COMMENTED** [review]
Sorry, should've caught this earlier

File: blog/docs/releases/announcing-knative-v1-1-release.md (hunk 12 above)
Comment (csantanapr) [reply blog/docs/releases/announcing-knative-v1-1-release.md:63]: Good catch, I just copy paste the link from the release notes
I will run a link checker on the whole file

**Review by snneji — CHANGES_REQUESTED** [review]

File: blog/docs/releases/announcing-knative-v1-1-release.md (hunk 12 above)
Comment (csantanapr) [reply blog/docs/releases/announcing-knative-v1-1-release.md]: This is fix now @psschwei

File: blog/docs/releases/announcing-knative-v1-1-release.md [hunk 13]
```diff
@@ -0,0 +1,166 @@
...
+
+#### RabbitMQ Broker and Source v1.1
+
+<!-- Original notes are here: https://github.com/knative-sandbox/eventing-rabbitmq/releases/tag/knative-v1.1.0 -->
+
+
+#### 💫 New Features & Changes
+
+- Adds tracing with opencensus to the dispatcher and ingress ([#370](https://github.com/knative-sandbox/eventing-rabbitmq/pull/370)
+- RabbitMQ source now can use predeclared queues instead of creating new ones ([#493](https://github.com/knative-sandbox/eventing-rabbitmq/pull/493)
```
[the remark this answers is not in the document]
Comment (csantanapr) [reply blog/docs/releases/announcing-knative-v1-1-release.md]: I preferred "predefined"

File: blog/docs/releases/announcing-knative-v1-1-release.md [hunk 14]
```diff
@@ -0,0 +1,172 @@
...
+- Adds tracing with opencensus to the dispatcher and ingress ([#370](https://github.com/knative-sandbox/eventing-rabbitmq/pull/370))
+- Adds tracing with opencensus to the dispatcher and ingress ([#370](https://github.com/knative-sandbox/eventing-rabbitmq/pull/370))
+- RabbitMQ source now can use predeclared queues instead of creating new ones ([#493](https://github.com/knative-sandbox/eventing-rabbitmq/pull/493))
+- Adds tracing with opencensus to the dispatcher and ingress ([#370](https://github.com/knative-sandbox/eventing-rabbitmq/pull/370))
+- Now the source Adapter processes messages concurrently depending on the channel_config.prefetch_count argument (default to 1) ([#522](https://github.com/knative-sandbox/eventing-rabbitmq/pull/522))
+- Adds tracing with opencensus to the dispatcher and ingress ([#370](https://github.com/knative-sandbox/eventing-rabbitmq/pull/370))
+- Do not expose the broker class that the controller operates on as an environment variable ([#512](https://github.com/knative-sandbox/eventing-rabbitmq/pull/512))
+- Adds tracing with opencensus to the dispatcher and ingress ([#370](https://github.com/knative-sandbox/eventing-rabbitmq/pull/370))
```
[the remark this answers is not in the document]
Comment (csantanapr) [reply blog/docs/releases/announcing-knative-v1-1-release.md:108]: I can't explain what happened here

**Comment by csantanapr** [comment]
@snneji thank you for the extensive review ❤️  PTAL

File: blog/docs/releases/announcing-knative-v1-1-release.md [hunk 15]
```diff
@@ -0,0 +1,166 @@
...
+
+<!-- Original notes are here: https://github.com/knative/operator/releases/tag/knative-v1.1.0   -->
+
+#### 💫 New Features & Changes
+
+- Refactor image transformer to de-duplicate and simplify it ([#863](https://github.com/knative/operator/pull/863))
+
+### Thank you, contributors
+
+Release leads: ([@matzew](https://github.com/matzew) and [@nak3](https://github.com/nak3))
```
Comment (snneji) [inline blog/docs/releases/announcing-knative-v1-1-release.md:134]:
```suggestion
Release leads: [@matzew](https://github.com/matzew) and [@nak3](https://github.com/nak3)

```

**Review by snneji — COMMENTED** [review]
Just one more formatting fix otherwise looks good

File: blog/docs/releases/announcing-knative-v1-1-release.md (hunk 15 above)
Comment (snneji) [reply blog/docs/releases/announcing-knative-v1-1-release.md:134]: Add extra line to fix bulleted list below

**Review by snneji — APPROVED** [review]

**Comment by snneji** [comment]
Thanks @csantanapr! 

/lgtm

**Pull request merged**